cmd/meroxa/root/apps/deploy.go

```diff
@@ -424,7 +424,7 @@ func (d *Deploy) buildApp(ctx context.Context) error {
 		d.logger.StopSpinner("\t")
```

Comment:
what about these... would we want a red X in these cases?

Reply:
could be part of an enhancement ticket

Reply:
@janelletavares great catch. I think we do. Updating...